### DISCUSSION TITLE
Shared video/image url has own landing page

### DISCUSSION THREAD

**Discussion by danshaw** [diced/zipline#375] [Ideas]
One of the main reasons for trying out Zipline is for the many times that I will record a screencast for friends/family. It would be great if I could upload a video/image and have the UI show the media on its own landing page with the metadata below it. Similar to what it looks like when previewing a video when logged in. Right now it looks like the URL is the .mp4 or .jpg file itself.

The video player could then use something like Plyr - https://plyr.io/. This is a beautiful player that allows for speeding up along with adjusting quality on the fly. I don't believe the player with Zipline has the ability to transcode or adjust quality on the fly depending on a users bandwidth? Maybe that is a server limitation?

A good example of a product that does something similar is DropShare. They call them landing pages. I'm attaching an example.  
<img width="1144" alt="preview" src="https://user-images.githubusercontent.com/6180024/232349197-eafd03b6-37c2-4ace-aad0-ec753c542907.png">

**Comment by diced** [comment]
You can use the embedded option to make the page not directly serve the raw file, though the other stuff like the name, date, etc., will not show up as of now. I think it could be a great addition, so I will think of adding it soon.

**Comment by diced** [comment]
this has been somewhat added in v4, pretty similar to the screenshot in this discussion as well. Closing!
